001-core/constants.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Automattic\VIP\Core\Constants;
+
+/**
+ * We use HyperDB, so DB_* constants are not automatically defined.
+ * We define them here in case any customer code is referencing them to avoid fatals errors in PHP 8+.
+ * As a best practice, these constants should not be used directly (`_doing_it_wrong()`).
```

Comment:
Give a more accurate Summary (first line of a DocBlock), and let the rest be the Description.

```suggestion
 * Define DB_HOST, DB_USER, DB_PASSWORD, and DB_NAME constants.
 *
 * WPVIP uses use HyperDB, so DB_* constants are not automatically defined.
 * As a best practice, these constants should not be used directly, but we define them here in
 * case any customer code does to avoid fatal errors in PHP 8+.
```